doc/src/explore/devnet.md

```diff
@@ -0,0 +1,189 @@
+* A [Discord channel](https://discordapp.com/channels/916379725201563759/971488439931392130) for requesting test coins that can be used to pay for gas on the test network. These coins have no financial value and will disappear each time we reset the network.
+
+Many improvements to the Sui DevNet are underway, such as the ability to run full nodes and use a browser-based wallet. See the [Sui DevNet blog post](https://medium.com/mysten-labs) announcement for full details on upcoming features.
+
+## Components
+
+DevNet deployment features the following components of the Sui blockchain:
+
+* The [Sui Wallet command line interface (CLI)](../build/wallet.md)
+    * create and manage your private keys
+    * submit transactions for creating example NFTs
+    * call and publish Move modules
+* [Sui Explorer](https://github.com/MystenLabs/sui/tree/main/explorer/client#readme) - view transactions and objects
+
+## Prerequisites
+
+### Set up environment
+
+To use the Sui DevNet, you will need:
+
+1. Sui test coins (tokens) requested through [Discord](https://discordapp.com/channels/916379725201563759/971488439931392130)
+1. A command line terminal, as virtually everything done here is done by CLI
+1. the [`git` command line interface](https://git-scm.com/download/)
+1. The [Rust and Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html) toolchain, updated with `rustup update stable`
+1. [Sui binaries](../build/install.md#binaries) in your PATH environment variable, particularly `wallet`
+
+Remember, you can confirm the existence of a command in your PATH by running `which` followed by the command, for example:
+
+```shell
+$ which wallet
+```
+You should see the path to the command. Otherwise, reinstall.
+
+> **Tip:** To reliably test DevNet with the latest Sui binaries, re-install them at least weekly.
+
+In addition, to conduct advanced work such as publishing a Move module or making a Move call, also obtain:
+
+1. a [GitHub account](https://github.com/signup) if you don't have one already
+1. the [Sui source code](../build/install.md#source-code); for simplicity, we recommend installing in `~/sui` or using an environment variable
+
+### Set up wallet, connect to gateway
+
+Now [set up your wallet and connect to DevNet](../build/wallet.md#connect-to-devnet) in a single step. Note you can [manually change the Gateway URL](../build/wallet.md#manually-changing-the-gateway-url) if you have already configured a Sui wallet.
+
+> **Tip:** If you run into issues, reset the Sui configuration by removing its directory, by default located at `~/.sui/sui_config`. Then reinstall [Sui binaries](../build/install.md#binaries).
+
+## Basic testing
+
+### Request gas tokens
+
+Request gas tokens in the [#request-devnet-test-coins](https://discordapp.com/channels/916379725201563759/971488439931392130) Discord channel if you haven't already and wait for them to be issued.
```

Comment:
We probably need to add something like the following to this paragraph @sblackshear 

You can request gas tokens by posting your wallet address in the the [#request-devnet-test-coins](https://discordapp.com/channels/916379725201563759/971488439931392130) Discord channel.

You can find your wallet address through the following command

```
$ wallet active-address
F16A5AEDCDF9F2A9C2BD0F077279EC3D5FF0DFEE
```

Reply:
Let's add that when you all figure it out. Proceeding with what we know now.

Reply:
Done